### PR TITLE
EES-1186 Add styles to ensure AccordionSection contents lazy load

### PR DESCRIPTION
**Pull request by markhiveit**
When using the govuk styles alone, content inside the AccordionSection
wouldn't lazy load until the user had interacted with the page - specifically tables and charts. This meant tables and charts would load forever until the user scrolled down the page a little. Using these additional AccordionSection
styles resolves this issue.

### CODE REVIEW

File: src/explore-education-statistics-common/src/components/AccordionSection.tsx [hunk 1]
```diff
@@ -103,7 +104,13 @@ const AccordionSection = ({
       </div>
       <div
         aria-labelledby={headingId}
-        className={classNames(classes.sectionContent)}
+        className={classNames(
+          classes.sectionContent,
+          styles.sectionContent,
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/components/AccordionSection.tsx:109]: Think `sectionContent` could just be applied on mount. I would expect `sectionContent` to essentially mirror what `govuk-accordion__section-content` does i.e. hide the content, but using `visibility: hidden` for the lazy loading.

As it doesn't get applied until mount, the non-JS version's content should be visible still.

File: src/explore-education-statistics-common/src/components/AccordionSection.tsx [hunk 2]
```diff
@@ -103,7 +104,13 @@ const AccordionSection = ({
...
+          onMounted({
+            [styles.sectionContentCollapsed]: !open,
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/components/AccordionSection.tsx:111]: If we change the `sectionContent` as suggested above, then this should be changed to `sectionContentExpanded` (and adjusted accordingly)

**Review by ntsim — CHANGES_REQUESTED** [review]

File: src/explore-education-statistics-common/src/components/AccordionSection.tsx (hunk 2 above)
Comment (markhiveit) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: I've done it slightly differently. I've kept this line and removed `.sectionContent` completely - and now I'm going to hope you don't tell me that will break lazy loading. I tried testing it locally and it seemed fine.

Comment (ntsim) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: Looks fine, but run the tests and find out :wink:

Comment (markhiveit) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: I updated the snapshots so I'm hoping that's all good

Comment (ntsim) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: I meant UI tests

Comment (markhiveit) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: I ran all the public tests and create_data_block_with_chart and everything passed. But create_data_block_with_chart was previously passing locally before - the failure was only happening on the pipeline.

**Review by ntsim — APPROVED** [review]

File: src/explore-education-statistics-common/src/components/AccordionSection.tsx (hunk 2 above)
Comment (ntsim) [reply src/explore-education-statistics-common/src/components/AccordionSection.tsx]: Cool, all good then :+1:

**Pull request merged**